dm/tests/print_status/run.sh

```diff
@@ -19,9 +19,12 @@ function run() {
 		"github.com/pingcap/tiflow/dm/syncer/ProcessBinlogSlowDown=sleep(4)")
 	export GO_FAILPOINTS="$(join_string \; ${inject_points[@]})"
 
+	cp $cur/conf/dm-worker1.toml $WORK_DIR/dm-worker1.toml
+	sed -i "s%placeholder%$WORK_DIR/relay_by_worker%g" $WORK_DIR/dm-worker1.toml
```

Comment:
maybe add this test to `relay it test`, add this in `print_status` is little not intuitive